conjur/api.py

```diff
@@ -48,8 +50,8 @@ def __init__(self,
         self._ca_bundle = ca_bundle
 
         self._account = account
-        if not self._account:
-            raise RuntimeError("Account cannot be empty!")
+        # if not self._account:
```

Comment:
Why commented code and not remoive it ?

Reply:
Not sure I wanted to touch that area yet because it didn't directly relate to my work with init. Decided against changing and uncommented